grails-app/views/consentGroupDocuments/_index.gsp

```diff
@@ -0,0 +1,24 @@
+<g:javascript>
+    const user = {
+        <g:if test="${session?.user}">
+            displayName: "${session.user.displayName}",
+            email: "${session.user.emailAddress}",
+            userName: "${session.user.userName}"
+        </g:if>
+    };
```

Comment:
See https://github.com/broadinstitute/orsp-pub/pull/33#discussion_r248318817

I think this approach is not the best one to use. The point of `const component` is that it is just a holder for constant values. We should instead pass in a URL that the component can call to get the current user. In the long term, these `const component` values will be rolled up into a single configuration component that the app has access to.